src/lexicon.js

```diff
@@ -8,16 +8,14 @@ window.gcexports.globalLexicon = {
   "rounded-xl" : { "tk": 1, "cls": "string", "val": "rounded-xl", "length": 0, "arity": 0 },
   "rounded-xxl" : { "tk": 1, "cls": "string", "val": "rounded-2xl", "length": 0, "arity": 0 },
   "rounded-xxxl" : { "tk": 1, "cls": "string", "val": "rounded-3xl", "length": 0, "arity": 0 },
-  "border-2" : { "tk": 1, "cls": "string", "val": "border-2", "length": 0, "arity": 0 },
-  "border-4" : { "tk": 1, "cls": "string", "val": "border-4", "length": 0, "arity": 0 },
-  "border-6" : { "tk": 1, "cls": "string", "val": "border-6", "length": 0, "arity": 0 },
-  "border-8" : { "tk": 1, "cls": "string", "val": "border-8", "length": 0, "arity": 0 },
+  "rounded-full" : { "tk": 1, "cls": "string", "val": "rounded-full", "length": 0, "arity": 0 },
   "border-amber" : { "tk": 1, "cls": "string", "val": "border-amber-600", "length": 0, "arity": 0 },
```

Comment:
If you change `"border-amber"` to `"amber"` and `"border-amber-600"` to `"amber-600"`, then your compiler code will do the right thing when you write: 
```
border [amber] button foo..
```
Same for the other colors.